src/components/Decks/containers/Decks.tsx

```diff
@@ -1,12 +1,12 @@
 import { Card, Decks as DecksType } from 'models/Deck'
 import DecksComponent from '../components/DecksComponent'
 
-const card1: Card = { id: '1', japanese_value: 'あ', romanji_value: 'a' }
-const card2: Card = { id: '1', japanese_value: 'あ', romanji_value: 'a' }
-const card3: Card = { id: '1', japanese_value: 'あ', romanji_value: 'a' }
-const card4: Card = { id: '1', japanese_value: 'あ', romanji_value: 'a' }
-const card5: Card = { id: '1', japanese_value: 'あ', romanji_value: 'a' }
-const card6: Card = { id: '1', japanese_value: 'あ', romanji_value: 'a' }
+const card1: Card = { id: '1', japanese_value: 'あ', romanji_value: 'a', reverse: false }
```

Comment:
This is ok, but you can try to use the already declared array of `hiragana_deck` to avoid declaring so many cards with unfunctional names. It's still WIP so it's not mandatory, but it's highly recommended